packages/voila/package.json

```diff
@@ -23,6 +23,7 @@
     "@jupyterlab/markdownviewer-extension": "^4.0.0",
     "@jupyterlab/markedparser-extension": "^4.0.0",
     "@jupyterlab/mathjax-extension": "^4.0.0",
+    "@jupyterlab/mathjax2-extension": "^4.0.0-alpha.21",
```

Comment:
Do we need mathjax-extension AND mathjax2-extension?

Reply:
Nope, only the latter is enough.